string label
geometry_msgs/Pose2D[] pose_furniture
